app/models/hmcts_common_platform/defendant_court_application_summary.rb

```diff
@@ -0,0 +1,50 @@
+module HmctsCommonPlatform
+  class DefendantCourtApplicationSummary
```

Comment:
I still don't see the value of creating a new class here - HmctsCommonPlatform::ApplicationSummary was designed explicitly to generate a response payload for the court application summaries in the prosecution cases endpoint. It literally has no other purpose than to do what you need it to do, so why not use it?